.buildkite/pipeline.yml

```
@@ -0,0 +1,19 @@
+agents:
+  queue: k8s-builders
+
+steps:
+  - label: make lint
+    key: lint
+    command: make lint
+    plugins:
+      - seek-oss/aws-sm#v2.3.2:
+          json-to-env:
+            - secret-id: sdlc/prod/buildkite/integration_provider_secret_redpanda_client
+  - label: make unit
+    key: unit
+    command: make unit
+    plugins:
+      - seek-oss/aws-sm#v2.3.2:
+          json-to-env:
+            - json-key: .
+              secret-id: devex/prod/buildkite/integration_provider_secret
```

```suggestion
            - secret-id: sdlc/prod/buildkite/integration_provider_secret_redpanda_client
```
make changes similar to the lines above in this file